### PR TITLE
deployment/script: Add a script that implements the prerequisites for deploying CCNP

### PR DESCRIPTION
**Pull request by Yanbo0101**

### CODE REVIEW

File: deployment/script/prerequisite.sh [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    #install helm
+    apt-get install -y helm
+}
+
+function install_pip {
+    #install python3-pip
+    apt install -y python3-pip 
+}
+
+function ccnp_udev_rules {
```
Comment (ruomengh) [inline deployment/script/prerequisite.sh:74]: You may want to check whether the files exist before running this function.

File: deployment/script/prerequisite.sh [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    else
+        echo "Error: No TDX deivce node with msg: $OUTPUT"
+        exit 1
+    fi
+
+    # make the udev setup effective
+    udevadm trigger
+}
+
+function ccnp_uds_dir {
```
Comment (ruomengh) [inline deployment/script/prerequisite.sh:94]: The same as above comments.

**Comment by ruomengh** [comment]
Cloud you please also add the script link to deployment guide?

File: deployment/script/prerequisite.sh (hunk 1 above)
Comment (Yanbo0101) [reply deployment/script/prerequisite.sh:86]: Fixed. Thanks

**Comment by Yanbo0101** [comment]
> Cloud you please also add the script link to deployment guide?

Updated, please review. Thanks.

**Review by ruomengh — APPROVED** [review]
LGTM

**Pull request merged**